pkg/adapter/apiserver/adapter_injection.go

```diff
@@ -65,14 +65,20 @@ func NewAdapter(ctx context.Context, processed adapter.EnvConfigAccessor, ceClie
 	if err := json.Unmarshal([]byte(env.ConfigJson), &config); err != nil {
 		panic("failed to create config from json")
 	}
-
+	ceOverridesExtensions := make(map[string]string)
```

Comment:
There is a real stuct that should be used for the ceOverrides, it is part of the source duck type

Reply:
Do you mean I should use this method https://github.com/knative/eventing/blob/a6fc5408cb4497afd1ace2432ee90d621d7f04ea/pkg/adapter/v2/config.go#L190 instead of writing it by myself?